BackendApartmentReservation/BackendApartmentReservation/Controllers/EmployeesController.cs

```diff
@@ -47,5 +49,26 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
             };
             return Ok(response);
         }
+
+        [HttpGet]
+        [Route("employees/{userId}")]
+        public async Task<IActionResult> GetEmployeeById(string employeeID)
+        {
+            try
+            {
+                var result = await _employeeManager.GetEmployeeById(employeeID);
+                var response = new EmployeeInfo()
```

Comment:
I would suggest to project DTO in manager, but when I think about it more, I think its kind of optional. What is your opinion?

Reply:
And wait, why are you creating the same `EmployeeInfo` type, just without id? It is a must to return an ID